public/electron.js

```diff
@@ -341,6 +344,42 @@ app.on('ready', async () => {
     }
   });
 
+  // called when getting geolocation enabled
+  ipcMain.on(GET_GEOLOCATION_ENABLED_CHANNEL, () => {
```

Comment:
Can we factor out the listener to a `listeners/getGeolocationEnabled` file? Have a look at the other files that have been factored out.